src/test/ui/suggestions/str-as-char.rs

```diff
@@ -8,6 +8,8 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// run-rustfix
```

Comment:
Can we detect this by the existence of a .fixed file?

Reply:
Ah I commented to this [above as well](https://github.com/rust-lang/rust/pull/50084#issuecomment-386401606)